app/controllers/batch_connect/session_contexts_controller.rb

```diff
@@ -22,16 +15,12 @@ def new
         #{@app.validation_reason} Please contact support if you see this message
       EOT
     end
-  rescue BatchConnect::App::AppNotFound => e
```

Comment:
Actually, these errors will only appear if the batch connect app is incorrectly built. An incorrectly built batch connect app should not be deployed to production. Unless the user is trying to access an app that does not exist.